docs/usage/shoot_credentials_rotation.md

```diff
@@ -167,8 +167,39 @@ However, this feature gate is deprecated, turned off by default and will be remo
 
 ## ETCD Encryption Key
 
-This key is used to encrypt the data of `Secret` resources inside etcd.
-It is currently **not** rotated automatically and there is no way to trigger it manually.
+This key is used to encrypt the data of `Secret` resources inside etcd (see [upstream Kubernetes documentation](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/)).
+
+The encryption key has no expiration date.
+There is no automatic rotation and **it is the responsibility of the end-user to regularly rotate the encryption key.**
+
+The rotation happens in three stages:
+
+- In stage one, a new encryption key is created and added to the bundle (together with the old encryption key).
+- In stage two, all `Secret`s in the cluster are rewritten by the `kube-apiserver` so that they become encrypted with the new encryption key.
+- In stage three, the old encryption is dropped from the bundle.
+
+Technically, the `Preparing` phase indicates the stages one and two.
+Once it is completed, the `Prepared` phase indicates readiness for stage three.
+The `Completing` phase indicates stage three, and the `Completed` phase states that the rotation process has finished.
+
+> You can check the `.status.credentials.rotation.etcdEncryptionKey` field in the `Shoot` to see when the rotation was last initiated, last completed, and in which phase it currently is.
+
+In order to start the rotation (stage one), you have to annotate the shoot with the `rotate-etcd-encryption-key-start` operation:
+
+```bash
+kubectl -n <shoot-namespace> annotate shoot <shoot-name> gardener.cloud/operation=rotate-etcd-encryption-key-start
```

Comment:
I executed 2 times the rotation in a row. I noticed that old `kube-apiserver-etcd-encryption-configuration` Secrets are not cleaned up:

```
kube-apiserver-etcd-encryption-configuration-10ebf368             Opaque                                1      108s
kube-apiserver-etcd-encryption-configuration-15b6cf29             Opaque                                1      11m
kube-apiserver-etcd-encryption-configuration-671b6f54             Opaque                                1      5m39s
kube-apiserver-etcd-encryption-configuration-6a3e72c0             Opaque                                1      23h
kube-apiserver-etcd-encryption-configuration-6dc4e190             Opaque                                1      22m
kube-apiserver-etcd-encryption-configuration-9e17b9c8             Opaque                                1      7m1s
kube-apiserver-etcd-encryption-configuration-a884d738             Opaque                                1      24m
```

Reply:
Yes, those will only be cleaned up every `12h` by the [garbage collector for immutable `ConfigMap`s/`Secret`s](https://github.com/gardener/gardener/blob/master/docs/concepts/resource-manager.md#garbage-collector-for-immutable-configmapssecrets): https://github.com/gardener/gardener/blob/af04d906c0a42a6b2dfd5923f21c0c493721ca2b/pkg/operation/botanist/component/resourcemanager/resource_manager.go#L700